src/and-cli-ls.ts

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+
+import { CommandRunner } from "./modules/command-runner";
+import { CommandDefinitions } from "./modules/command-definitions";
+import program from "commander";
+import { Options } from "./constants/options";
+import { ListCommands } from "./modules/list-commands";
+
+// -----------------------------------------------------------------------------------------
+// #region Constants
+// -----------------------------------------------------------------------------------------
+
+const { DEFAULT_OPTIONS } = ListCommands;
+const {
+    includeHelp: DEFAULT_INCLUDE_HELP,
+    indent: DEFAULT_INDENT,
+    prefix: DEFAULT_PREFIX,
+    skipCache: DEFAULT_SKIP_CACHE,
+    useColor: DEFAULT_USE_COLOR,
+} = DEFAULT_OPTIONS;
+
+// #endregion Constants
+
+CommandRunner.run(async () => {
+    // -----------------------------------------------------------------------------------------
+    // #region Entrypoint
+    // -----------------------------------------------------------------------------------------
+
+    program
+        .description(CommandDefinitions.ls.description)
+        .option(
+            "-i, --indent <indent>",
```

Comment:
I am not sure of the standard; should this be `-i <indent>, --indent <indent>` (assuming that `-i` doesn't have a default indent)?